includes/features/SU-SWS/stanford_courses/stanford_courses.feature

```diff
@@ -4,9 +4,18 @@ Feature: Stanford Courses
   As an administrative user
   I want to ensure that the Stanford Courses module is working properly
 
+
   Background:
     Given the "stanford_courses" module is enabled
 
+  @api
```

Comment:
This is the same thing as the background task. We could change this to test to see if the checkbox is enabled.